### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by mesparza5**
This PR adds new updates for the UI look of the Dashboard. 

Ref Issue #8471

### CODE REVIEW

File: assets/cms/scss/_panels.scss [hunk 1]
```diff
@@ -1 +1,2 @@
 @import "panels/shared";
+//@import "panels/base";
```
Comment (KorvinSzanto) [inline assets/cms/scss/_panels.scss]: Do you mean to have this commented out?

**Comment by aembler** [comment]
Did some files accidentally get added back? You might want to check that some files haven’t been removed in the bedrock repo that you’re adding back.

File: assets/cms/scss/_panels.scss (hunk 1 above)
Comment (mesparza5) [reply assets/cms/scss/_panels.scss]: Yes, that's correct, we can remove it tho.

**Comment by mesparza5** [comment]
@aembler I was working on the panels and when you add your updates I just comment and leave what I was doing.

File: assets/cms/scss/_panels.scss (hunk 1 above)
Comment (mesparza5) [reply assets/cms/scss/_panels.scss]: This was corrected in the latest pull request.

File: assets/cms/scss/panels/_shared.scss [hunk 2]
```diff
@@ -366,6 +375,10 @@ html.ccm-panel-ready div.ccm-page.ccm-panel-detail-disable-transition {
   transition: none;
 }
 
+div.ccm-panel-dashboard-footer {
+  margin:20px 0;
```
Comment (KorvinSzanto) [inline assets/cms/scss/panels/_shared.scss:379]: Can you fix the formatting here

Comment (KorvinSzanto) [reply assets/cms/scss/panels/_shared.scss:379]: I'll just make this change post merge

Comment (KorvinSzanto) [reply assets/cms/scss/panels/_shared.scss:379]: https://github.com/concrete5/bedrock/commit/63a3a3cdc72bbf735b8af5931e8af18a68d4967f

Comment (mesparza5) [reply assets/cms/scss/panels/_shared.scss:379]: Thanks @KorvinSzanto !

**Pull request merged**